tests/baselines/reference/user/npm.log

```diff
@@ -30,6 +30,7 @@ node_modules/npm/bin/npm-cli.js(132,17): error TS2339: Property 'config' does no
 node_modules/npm/bin/npm-cli.js(134,17): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/bin/npm-cli.js(136,17): error TS2339: Property 'config' does not exist on type 'typeof EventEmitter'.
 node_modules/npm/html/static/toc.js(3,40): error TS2531: Object is possibly 'null'.
+node_modules/npm/html/static/toc.js(28,3): error TS2531: Object is possibly 'null'.
```

Comment:
`document.body: HTMLElement | null`. Was it just `HTMLElement` before?

Reply:
Yea.

Reply:
After discussion with @weswigham and @rbuckton, we are pretty sure that this should still be `HTMLElement`. `HTMLElement | null`is even more pedantic than making array indexing nullable. I'll file an issue on TSJS-libgenerator to track this fix.

@andy-ms once that fix is in, all the changes you had to make to DefinitelyTyped will be obsolete, sorry. :(

Reply:
Filed issue Microsoft/TSJS-lib-generator#558